dask/diagnostics/profile_visualize.py

```diff
@@ -169,16 +168,9 @@ def visualize(profilers, file_path=None, show=True, save=True, **kwargs):
     The completed bokeh plot object.
     """
     bp = import_required('bokeh.plotting', _BOKEH_MISSING_MSG)
-    import bokeh
+    from bokeh.io import state
 
-    if LooseVersion(bokeh.__version__) >= "0.12.10":
-        from bokeh.io import state
-        in_notebook = state.curstate().notebook
-    else:
-        from bokeh.io import _state
-        in_notebook = _state._notebook
-
-    if not in_notebook:
+    if not state.curstate().notebook:
```

Comment:
I think you mean to do:
```
from bokeh.io import curstate
```

and 
```
if not curstate().notebook:
```

Reply:
Also just to be completely clear, what `curstate().notebook` means is that `bokeh.io.output_notebook()` has been called, nothing else.

Reply:
Right, IIUC the intent of this code is to determine whether we should display the visualization in the notebook or in a new tab.